src/ui/public/inspector/inspector.tsx

```diff
@@ -102,19 +48,14 @@ interface InspectorOptions {
  * Opens the inspector panel for the given adapters and close any previously opened
  * inspector panel. The previously panel will be closed also if no new panel will be
  * opened (e.g. because of the passed adapters no view is available). You can use
- * {@link InspectorSession#close} on the return value to close that opened panel again.
+ * {@link FlyoutSession#close} on the return value to close that opened panel again.
  *
  * @param {object} adapters - An object of adapters for which you want to show
  *    the inspector panel.
  * @param {InspectorOptions} options - Options that configure the inspector. See InspectorOptions type.
- * @return {InspectorSession} The session instance for the opened inspector.
+ * @return {FlyoutSession} The session instance for the opened inspector.
  */
-function open(adapters: Adapters, options: InspectorOptions = {}): InspectorSession {
-  // If there is an active inspector session close it before opening a new one.
-  if (activeSession) {
-    activeSession.close();
-  }
-
+function open(adapters: Adapters, options: InspectorOptions = {}): FlyoutSession {
```

Comment:
If possible let's use `InspectorSession` here still, even if we just define it as `type InspectorSession = FlyoutSession` for now, but that way a consumer doesn't need to possibly change a type, if we want to add more specific methods for the inspector later on.